homeassistant/components/honeywell/climate.py

```diff
@@ -24,6 +25,8 @@
 
 _LOGGER = logging.getLogger(__name__)
 
+DOMAIN = 'honeywell'
```

Comment:
You can import the DOMAIN but don't set it on a platform

Reply:
see above